app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    text = command.split(" ")
```

Comment:
Consider using more descriptive variable names. Instead of `text`, you could use `command_parts` to make it clearer that this variable holds the parts of the command string.